community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/ExecutionContext.scala

```diff
@@ -78,32 +85,32 @@ case class MapExecutionContext(m: MutableMap[String, AnyValue])
     m.foreach(f)
   }
 
-  override def +=(kv: (String, AnyValue)) = {
+  override def +=(kv: (String, AnyValue)): MapExecutionContext.this.type = {
```

Comment:
These types are IntelliJ artifacts and not helpful/pretty. In this case, you should just use `MapExecutionContext`.

Reply:
Never mind me. ^ That was a lie.